Mixpanel/TakeoverNotificationViewController.swift

```diff
@@ -140,10 +140,22 @@ class TakeoverNotificationViewController: BaseNotificationViewController {
     }
 
     override func show(animated: Bool) {
-        window = UIWindow(frame: CGRect(x: 0,
-                                        y: 0,
-                                        width: UIScreen.main.bounds.size.width,
-                                        height: UIScreen.main.bounds.size.height))
+#if !BUILDING_FOR_APP_EXTENSION
+        if #available(iOS 13.0, *) {
```

Comment:
will this still work in iOS13 if they _aren't_ using scenes?

Reply:
yes, it works. iOS 13 replace the top level UIWindow to UIWindowScene regardless whether you use SceneDelegate or not.